packages/design-system/src/components/Form/Field/Input/Input.Copy.tsx

```diff
@@ -68,7 +48,6 @@ const InputCopy = forwardRef(
 			<FieldPrimitive
 				label={label}
 				description={getDescriptionMessage()}
-				hasError={!!copyError}
```

Comment:
As a user, you lost information that copy didn't work

Reply:
yes :disappointed: but I have not been able to reproduce. Do you have a use case ?
AFAIK we do not support browser that do not have the feature.